test/plugin/scenarios/dbcp-2.x-scenario/support-version.list

```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+2.7.0
+2.6.0
+2.5.0
+2.4.0
```

Comment:
Do DBCP have 2.0-2.4?

Reply:
> Do DBCP have 2.0-2.4?

@wu-sheng Thanks for your review~ Yes, sir.  It also has these versions
2.0 - 2.4: 2.3.0/2.2.0/2.1.1/2.1/2.0.1/2.0 
I've tested these versions locally. Do I need to add it to the list？

Reply:
We should add the latest of each version, 2.3.0, 2.2.0, 2.1.1, 2.0.1

Reply:
> We should add the latest of each version, 2.3.0, 2.2.0, 2.1.1, 2.0.1

Ok sir, I have updated the list.